NEWS.md

```diff
@@ -1,13 +1,13 @@
 
 # vctrs (development version)
 
-* `new_vctr()` now always appends a base `"list"` class to list `.data` to
-  be compatible with changes to `vec_is_list()`. This affects `new_list_of()`,
-  which now returns an object with a base class of `"list"`.
+* `new_vctr()` now defaults to `inherit_base_type = TRUE` for list `.data`, as
+  generally they should be considered lists by `vec_is_list()`.
 
 * `vec_is_list()` no longer allows S3 lists that implement a `vec_proxy()`
   method to automatically be considered lists. A S3 list must explicitly
-  inherit from `"list"` in the base class to be considered a list.
+  inherit from `"list"` or `"vctrs_list_of"` in the base class to be considered
```

Comment:
If this is just a stopgap, I don't think we should advertise the `vctrs_list_of` special case